apps/src/gamelab/GameLabGroup.js

```diff
@@ -76,13 +76,28 @@ module.exports.Group = function (baseConstructor) {
    * @return {boolean} True if any touching occurred
    */
   array.isTouching = function (target) {
-    var didTouch = false;
-    for (var i = 0; i < this.length; i++) {
-      didTouch = this[i].isTouching(target) || didTouch;
-      // Note: can't early out because each call needs to update the sprite's
-      // touching property.
+    var state = jsInterpreter.getCurrentState();
```

Comment:
Per offline discussion, this method shares duplicate code with _groupCollideGameLab which essentially implement an interpreter-friendly for loop. Ways to deal with this include:
1. extract a "forEach" utility method
2. make _groupCollideGameLab more general, so that it can call isTouching rather than only AABBops

per our discussion it seems fine to wait until this logic is needed a 3rd time, and then consider extracting forEach.